.yarnrc

```diff
@@ -0,0 +1,2 @@
+
+save-prefix false
```

Comment:
using `^` is generally bad idea because not every library is following semantic versioning. This comes from my experience, I like to have strict versions of any dependencies, so other devs are everytime on the same page ( before possible via strict version and npm shrinkwrap, nowadays strict version and yarn.lock`